scenario/elasticreload.go

```diff
@@ -60,8 +51,14 @@ func (settings *ElasticReloadSettings) UnmarshalJSON(arg []byte) error {
 		if deprecated.AppName != "" {
 			hasSettings = append(hasSettings, "appname")
 		}
+		if deprecated.PollInterval > 0 {
+			hasSettings = append(hasSettings, "pollinterval")
+		}
+		if deprecated.SaveLog {
+			hasSettings = append(hasSettings, "log")
+		}
 		if len(hasSettings) > 0 {
-			return errors.Errorf("%s settings<%s> are no longer used", ActionElasticReload, strings.Join(hasSettings, ","))
+			return errors.Errorf("%s settings<%s> are no longer used, remove this setting/-s from script", ActionElasticReload, strings.Join(hasSettings, ","))
```

Comment:
I think the common way to write this would be "setting(s)".

Maybe ", remove listed setting(s) from script" so that we don't have the this/these issue.

Reply:
mm true